docs/DeveloperGuide.md

```diff
@@ -118,13 +119,14 @@ The `UI` component,
 
 **API** : [`Logic.java`](../src/main/java/seedu/address/logic/Logic.java)
 
-1. `Logic` uses the `Parser` class to parse the user command.
-2. This results in a `Command` object which is executed by the `LogicManager`.
-3. The command execution can affect the `Model` (e.g. adding a person) and/or raise events.
-4. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
+1. `Logic` invokes the `Parser` class to parse the user command.
+2. Upon recognizing the command, it sends the arguments to the `CommandParser` class, which recognizes the remaining arguments.
+3. This information is then captured in a `Command` object which is executed by the `LogicManager`.
+4. The command execution can affect the `Model` (e.g. adding a person) and/or raise events.
```

Comment:
What do you mean here as person? Are you using user case for person or otherwise make sure all the naming are correct and match with your current product.